compiler/fir/checkers/src/org/jetbrains/kotlin/fir/analysis/checkers/declaration/FirSealedSupertypeChecker.kt

```diff
@@ -29,19 +31,37 @@ object FirSealedSupertypeChecker : FirClassChecker() {
     }
 
     private fun checkGlobalDeclaration(declaration: FirClass, context: CheckerContext, reporter: DiagnosticReporter) {
-        for (it in declaration.superTypeRefs) {
-            val classId = it.coneType.classId ?: continue
+        val subclassPackage = declaration.classId.packageFqName
+        val subClassSession = declaration.moduleData.session
+        for (superTypeRef in declaration.superTypeRefs) {
+            val classId = superTypeRef.coneType.classId ?: continue
 
             if (classId.isLocal) {
                 continue
             }
 
-            val classSymbol = context.session.symbolProvider.getClassLikeSymbolByFqName(classId) as? FirRegularClassSymbol ?: continue
+            val superClass = context.session.symbolProvider.getClassLikeSymbolByFqName(classId)
+                    as? FirRegularClassSymbol ?: continue
```

Comment:
(Super) Nit: yeah, this is cosmetic (and surprised IJ formatter doesn't bother with this), but it looks to me quite unusual line wrapping.

https://developer.android.com/kotlin/style-guide#where_to_break

It might be better to glue type casting together, and rather break at `?:` I think.

Reply:
Thanks! Sounds good.